src/main/java/org/gridsuite/explore/server/services/ExploreService.java

```diff
@@ -215,9 +215,13 @@ public void deleteElementsFromDirectory(List<UUID> uuids, UUID parentDirectoryUu
         }
     }
 
-    public void updateFilter(UUID id, String filter, String userId, String name) {
+    public void updateFilter(UUID id, String filter, String userId, String name, String description) {
         filterService.updateFilter(id, filter, userId);
-        updateElementName(id, name, userId);
+
+        ElementAttributes elementAttributes = new ElementAttributes();
+        elementAttributes.setDescription(description);
+        elementAttributes.setElementName(name);
```

Comment:
to protect name not empty, add check isBlank for name before setting into elementAttributes..